bin/main

```diff
@@ -1,3 +1,51 @@
 #!/usr/bin/env ruby
 
-puts "Hello World!"
\ No newline at end of file
+
+while true
```

Comment:
**[ OPTIONAL ]** this while loop is not implemented correctly one way to do this would be to set a variable for example `game_on = true` then start your loop: `while game_on` and then when a winning condition is met or in this case when you are finished with your puts instead of breaking out of the loop just set your `game_on` variable to false and it should break out. Also, it would look better.